docs/token-guides/wsteth-bridging-guide.md

```diff
@@ -0,0 +1,353 @@
+
+```mermaid
+graph TD;
+  A("Is wstETH already bridged and has got adoption?")
+  B("Is the token upgradable (meets R-4)?")
+  C("Follow the general scenario <br> towards DAO recognition")
+  D("Consider migrating liquidity and redeploying, <br> following the general scenario towards <br> DAO recognition")
+  F["Consider delivering and/or committing <br> to deliver the missing parts.<br>Contact NEW for the best way forward"]
+
+  A-- Yes -->B
+  B-- Yes -->F
+  A-- No -->C
+  B-- No -->D
+```
+
+## General scenario towards the Lido DAO recognition
+
+This section describes an approximate path to bridging wstETH to an L2 network. The order of the steps is not strict but follows the general flow.
+
+🐾 Study the bridging guide and fill in [the questionnaire](#questionnaire) about your solution and send it to the NEW.
+
+🐾 Coordinate on priority lane, timings, and reviews with the NEW.
+
+🐾 Get the architecture and the deploy configuration verified by the NEW.
+
+🐾 Deploy the contracts to testnet. Get the testnet deployment verified by coordinating through the NEW.
+
+🐾 Express intention to bridge wstETH on the forum, outlining the details and technical plan. Consider:
+- Target one network per proposal to make the discussion more focused.
+- The post should be published in advance, ideally at least two weeks before any potential snapshot vote, to allow time for discussion and verification of the proposal.
+- The deployment addresses are not required at once but must be proposed at least a week before the snapshot voting starts.
+- If the proposed solution does not satisfy some of the recommendations (**R5...**), consider including the roadmap and committing to deliver it.
+- Examples:
+    - [wstETH to Base](https://research.lido.fi/t/wsteth-deployment-to-base-and-ownership-acceptance-by-lido-dao/5668)
+    - [wstETH to zkSync Era](https://research.lido.fi/t/wsteth-deployment-on-zksync/5701)
+    - [wstETH to Mantle](https://research.lido.fi/t/wsteth-deployment-on-mantle/5991)
+    - [wstETH to Linea](https://research.lido.fi/t/wsteth-on-linea-ownership-acceptance-by-lido-dao/5961)
+
+🐾 Deploy the contracts to mainnet. Get the mainnet deployment verified by the Lido protocol contributors and external security group if applicable (coordinating through NEW).
+
+🐾 Pass snapshot voting on https://snapshot.org/#/lido-snapshot.eth/. It should contain the final mainnet addresses and audits according to **R1**. Otherwise, one more snapshot voting with the addresses would have been required.
+
+## Recommendations
+
+This section enumerates design and security recommendations for a wstETH bridging solution.
+
+### Security and future-proof baseline
+
+The baseline recommendations: if any are not met there is a minimal chance of the Lido DAO recognition or NEW acknowledgment.
+
+#### R-1: Audited code and verifiable deployment
+
+The entire codebase (rollup, bridge, token) must be audited by a reputable third party.
```

Comment:
Rollup? 0_0 Does any rollup support continuous audits for their codebase? Its usually pretty costly.